helm-chart/renku/templates/ingress.yaml

```diff
@@ -59,53 +53,54 @@ spec:
             port:
               name: {{ $keycloakServicePort }}
       {{- end }}
-      - path: /gitlab
+      - path: /api
         pathType: Prefix
         backend:
           service:
             name: {{ template "renku.fullname" $ }}-gateway
             port:
               number: 80
-      - path: /repos
+      - path: /
         pathType: Prefix
         backend:
           service:
-            name: {{ template "renku.fullname" $ }}-gateway
+            name: {{ $uiFullname }}
             port:
-              number: 80
-      - path: /api
+              number: {{ $uiServicePort }}
+      - path: /ui-server
         pathType: Prefix
         backend:
           service:
             name: {{ template "renku.fullname" $ }}-gateway
             port:
-              number: 80
-      - path: /entities
```

Comment:
Moved further down in the file in an if statement that templates this only when v1 services are enabled.